i2c-ch341.c

```diff
@@ -36,9 +40,29 @@
 /* Limit the transfer read size in one go to 32 bytes. */
 #define MAX_RD_LENGTH 32
 
+/*
+ * Pin configuration. The I2C interfaces use 2 pins
+ * allocated to GPIOs. But we need to grab 4 pins
+ * to make sure that we don't mix signals.
+ */
```

Comment:
Maybe the gpio driver should use the new method if i2c isn't loaded/used and use the previous method if it is.

Reply:
I think I made a missleading comment here. We can still use the new method as it will still have the advantage to have more pins availabe without any interfering. The only problem are the last 4 GPIOs.
Do you prefer a check that the driver is not loaded over the is the line used in the gpio code? This would have the advantage that we might not need to change the i2c driver at all. But the disadvantage that the amount of exported GPIOs differs. And what should happen if one of the GPIO lines is already used? I'm not sure what's easier to understand for the user.
But it might be a good first solution to get this PR done.